sdk/guestconfiguration/Microsoft.Azure.Management.GuestConfiguration/src/Properties/AssemblyInfo.cs

```diff
@@ -7,8 +7,8 @@
 [assembly: AssemblyTitle("Microsoft Azure GuestConfiguration Management Library")]
 [assembly: AssemblyDescription("Provides Microsoft Azure Guest Configuration management operations including the ability to create, update and get guest configuration assignments, compliance statuses, guest configuration assignment reports.")]
 
-[assembly: AssemblyVersion("1.5.0")]
-[assembly: AssemblyFileVersion("1.5.0")]
+[assembly: AssemblyVersion("1.6.0")]
```

Comment:
As interface changes caused not backward compatible, please increate the major version of the package, not minor version

Reply:
done